### DISCUSSION TITLE
Remember volume mixer window position

### DISCUSSION THREAD

**Discussion by MrCrowCZ** [File-New-Project/EarTrumpet#706] [Ideas]
Is there any chance to open "volume mixer" on second monitor or save position some how?

**Comment by eender** [comment]
+1 SO MUCH.
Please remember the volume set for a specific app. For example the game "Modern Warfare" is back to 100% volume after a restart. Its always a guaranteed jumpscare and blows out my ears.

**Comment by elMooXos** [comment]
I would love that feature, came to just ask about it ^^

**Comment by TwoCables** [comment]
I'm a bit late to this, but you can use FancyZones in Microsoft PowerToys. I discovered this today and it works beautifully, once configured of course.
